backend/ticketvise/models/comment.py

```diff
@@ -71,11 +71,14 @@ def save(self, force_insert=False, force_update=False, using=None, update_fields
 
         if self.is_reply:
             if self.ticket.status == "CLSD":
-                if self.author.id == self.ticket.author.id:
-                    self.ticket.status = "ASGD"
+                if self.author.is_assistant_or_coordinator(self.ticket.inbox):
+                    if self.ticket.assignee:
+                        self.ticket.status = "ANSD"
+                    else:
+                        self.ticket.status = "PNDG"
                 else:
-                    self.ticket.status = "ANSD"
-            elif self.author.id != self.ticket.author.id:
+                    self.ticket.status = "ASGD"
+            elif self.author.is_assistant_or_coordinator(self.ticket.inbox):
                 self.ticket.status = "ANSD"
 
```

Comment:
Heb je getest wat er gebeurt als een student 2x een reply stuurt als er nog geen reply van een staff member is?

Reply:
Wat zou het probleem zijn als dit zo is? de status zou dan niet veranderen